### PR TITLE
updated readme and webview url

### PR DESCRIPTION
**Pull request by caspahouzer**
replaces references from appcelerator.com to titaniumsdk.com

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-   Copyright 2016 by Appcelerator
+   Copyright 2021 by TiDev
```
Comment (cb1kenobi) [inline LICENSE:1]: It's 2022 now! :)

File: README.md [hunk 2]
```diff
@@ -18,35 +18,26 @@ It also includes full support for iOS, Android & Windows.
...
+Code contributions are greatly appreciated, please submit a new [pull request](https://github.com/tidev/kitchensink-v2/pull/new/master)!
 
-### Legal Stuff
+<!-- ### Legal Stuff
 
 Appcelerator is a registered trademark of Appcelerator, Inc. Titanium is
-a registered trademark of Appcelerator, Inc.  Please see the LEGAL information about using our trademarks,
-privacy policy, terms of usage and other legal information at [http://www.appcelerator.com/legal](http://www.appcelerator.com/legal).
+a registered trademark of Appcelerator, Inc. Please see the LEGAL information about using our trademarks,
+privacy policy, terms of usage and other legal information at [http://www.appcelerator.com/legal](http://www.appcelerator.com/legal). -->
```
Comment (cb1kenobi) [inline README.md:43]: The legal link is dead. I can't find an analog under axway.com. This whole legal stuff section could probably be removed.

File: app/controllers/phone/map.js [hunk 3]
```diff
@@ -20,9 +20,8 @@ function handleMapClick(e) {
 }
 
 /* Right now Alloy doesn't add circles/polylines/polygons to map properly when they're created in Alloy
- * See https://jira.appcelerator.org/browse/ALOY-1608 for more info
```
Comment (cb1kenobi) [inline app/controllers/phone/map.js:23]: JIRA is dead! Long live JIRA archive! https://jira-archive.titaniumsdk.com/ALOY/ALOY-1608

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (caspahouzer) [reply LICENSE]: Back to the future ;-)

File: README.md (hunk 2 above)
Comment (caspahouzer) [reply README.md]: removed now

File: app/controllers/phone/map.js (hunk 3 above)
Comment (caspahouzer) [reply app/controllers/phone/map.js:23]: linked to archive

**Review by caspahouzer — COMMENTED** [review]
Everything seems to be okay now

**Review by cb1kenobi — APPROVED** [review]
Looks good! **APPROVED**

**Pull request merged**